### PR TITLE
[Merged by Bors] - chore: forward-port leanprover-community/mathlib#19028

### PR DESCRIPTION
**Pull request by eric-wieser**
This was "feat(topology/metric_space): diameter of pointwise zero and addition"

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Topology/Bornology/Basic.lean
Comment (eric-wieser) [inline Mathlib/Topology/Bornology/Basic.lean:1]: This file has no SHA change since it was split from another file.

**Review by YaelDillies — APPROVED** [review]
Thanks. This is an accurate forward-port!

maintainer merge

**Comment by riccardobrasca** [comment]
Thanks!

bors d+

**Comment by eric-wieser** [comment]
bors merge

**Pull request closed without merging**